Assets/DeleteAfterTime.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DeleteAfterTime : MonoBehaviour
```

Comment:
does this need to subclass monobehaviour?

Reply:
Yes